analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.TryCatchFinally.cs

```diff
@@ -549,10 +549,14 @@ public void TryCatch_ThrowInTry_SingleCatchBlock_ReThrowException()
             validator.ValidateTagOrder(
                 "BeforeTry",
                 "InTry",
-                "InCatch",          // With Exception thrown by Tag("InTry")
-                "InCatch");         // With Exception thrown by `throw`
+                "InCatch", // With Exception thrown by Tag("InTry")
+                "InCatch"); // With Exception thrown by `throw`
 
-            ValidateHasOnlyUnknownExceptionAndSystemException(validator, "InCatch");
+            validator.TagStates("InCatch").Should().HaveCount(2)
+                .And.ContainSingle(x => HasSystemException(x))
+                .And.ContainSingle(x => HasUnknownException(x));
+
+            validator.ValidateExitReachCount(1);
```

Comment:
_Line 1228:_ ValidateHasOnlyUnknownExceptionAndSystemException
_Line 1233:_ ValidateHasOnlyNoExceptionAndUnknownException
These two methods are private and not at the end of the file - probably a rebase/merge result in previous PRs on master